controllers/users.js

```diff
@@ -462,6 +464,27 @@ const filterUsers = async (req, res) => {
   }
 };
 
+const migrate = async (req, res) => {
+  try {
+    // Fetch user data from GitHub API for each document in the users collection
+    const usersSnapshot = await firestore.collection("users").get();
+    for (const userDoc of usersSnapshot.docs) {
+      const username = userDoc.data().github_id;
+      const response = await axios.get(`https://api.github.com/users/${username}`);
+      const githubUserId = response.data.id;
+
+      // Update the user document with the GitHub user ID
+      await userDoc.ref.update({
+        github_user_id: githubUserId,
+      });
```

Comment:
If the field already exist then should we force update the document ?

Reply:
The field is not there, I've created one and then fetch the value from GitHub API.